EKF/control.cpp

```diff
@@ -535,6 +535,7 @@ void Ekf::controlGpsFusion()
 					if (resetGpsAntYaw()) {
 						_control_status.flags.yaw_align = true;
 						startGpsYawFusion();
+						_time_last_gps_yaw_fuse = _time_last_imu;
```

Comment:
Should `_time_last_gps_yaw_fuse` be better set inside the `resetGpsAntYaw`?
Maybe we could also come up with a better name for `resetGpsAntYaw`, e.g. `resetYawToGps` or `resetYawToDualGps`, ...

Reply:
It kind of makes sense to move it inside the reset function, but I followed the pattern used for the other resets e.g.: when we call `resetHorizontalPositionToGps`, we set `_time_last_gps = _time_last_imu;` just after. Not sure what's better.
https://github.com/PX4/ecl/blob/97225fcb7431ddcb2ca1e2f72ea79043597ebea6/EKF/control.cpp#L581-L593

Reply:
I see. I would argue when the resetFunction matches only to one last_sensor_fuse, then we should move it into the resetFunction. So nobody can forget to set the timestamp of the last fusion.

Reply:
I am not sure if your example above is suitable. `_time_last_gps_yaw_fuse` stores the timestamp of the last fusion event. 
`_time_last_gps` is the timestamp of the newest measurement in the buffer. I think it is wrong to update this `_time_last_gps` outside of the `EstimatorInterface::setGpsData` function in the first place. This seems to be a bug and should be removed.

Reply:
Apparently, we not update any of the [fusion timestamps](https://github.com/PX4/ecl/blob/master/EKF/ekf.h#L344-L352) always during a reset of the respective state. So should we even do it for the gps_yaw, or should we do it everywhere?

Reply:
>_time_last_gps is the timestamp of the newest measurement in the buffer. I think it is wrong to update this _time_last_gps outside of the EstimatorInterface::setGpsData function in the first place. This seems to be a bug and should be removed.

I agree, not sure why this is there

Reply:
>I see. I would argue when the resetFunction matches only to one last_sensor_fuse, then we should move it into the resetFunction. So nobody can forget to set the timestamp of the last fusion.

Ok, you convinced me, I moved it to the reset function. I was also not happy to have that independent gps yaw fusion control logic in the `controlGpsFusion` function so I extracted it into `controlGpsYawFusion`.